packages/flame_audio/example/lib/main.dart

```diff
@@ -29,7 +29,11 @@ class AudioGame extends FlameGame with TapDetector {
 
   @override
   Future<void> onLoad() async {
-    pool = await AudioPool.create('fire_2.mp3', minPlayers: 3, maxPlayers: 4);
+    pool = await FlameAudio.createPool(
+      'sfx/fire_2.mp3',
```

Comment:
`sfx` is no longer default right?

Reply:
no, I thought that was just complicating things. now they all use the same cache